GP-Engine-team/GP_Engine
# Added an icon for the editor

## Description
This is a simple addition: a logo for the editor.
This is the icon used:
![green_portal](https://user-images.githubusercontent.com/33126700/120921604-04b02e00-c6b4-11eb-9ace-0c33c874ad93.png)

## How to test
Open the editor, and check that the logo appears on the left of the top window bar, and in the bottom bar
